### ISSUE TITLE
Deprecate older/redundant styles

### ISSUE THREAD

**Issue by Stanton** [jadu/pulsar#352]
Stylesheets listed here are those suspected to be no longer required, or which can be replaced easily.

- [x] **_component.choice-matrix.scss**
Was used in Q/BECK at one point

- [x] **_component.decks.scss**
May be used by Spacecraft custom modules

- [x] **_component.label-groups.scss**
Used in CXM history view

- [x] **_component.richer-list.scss**
predecessor of piano list

- [x] **_component.summary.scss**
predecessor of ribbon component used in CXM

- [x] **_component.tour.scss**
prototype tour UI, possibly hopscotch, never used in production
- [x] **_component.uploader.scss**

- [x] **_component.widget.scss**
Used in old prototype UI

**Comment by Stanton** [comment]
Output from analyze-css

```"metrics":{  
      "base64Length":8483,
      "redundantBodySelectors":0,
      "redundantChildNodesSelectors":7,
      "colors":120,
      "comments":2,
      "commentsLength":252,
      "complexSelectors":198,
      "duplicatedSelectors":17,
      "duplicatedProperties":17,
      "emptyRules":0,
      "expressions":0,
      "oldIEFixes":6,
      "imports":2,
      "importants":53,
      "mediaQueries":82,
      "notMinified":1,
      "multiClassesSelectors":405,
      "parsingErrors":4,
      "oldPropertyPrefixes":31,
      "propertyResets":2,
      "qualifiedSelectors":87,
      "specificityIdAvg":0,
      "specificityIdTotal":0,
      "specificityClassAvg":2.15,
      "specificityClassTotal":5061,
      "specificityTagAvg":0.57,
      "specificityTagTotal":1334,
      "selectors":2357,
      "selectorLengthAvg":1.7619855748833262,
      "selectorsByAttribute":122,
      "selectorsByClass":3796,
      "selectorsById":0,
      "selectorsByPseudo":1425,
      "selectorsByTag":381,
      "length":261526,
      "rules":1581,
      "declarations":2917
   }```

**Comment by Stanton** [comment]
Closed by #360
